packages/charts/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -307,7 +307,12 @@ export class CartesianChartBase
         yMaxValue: this.props.yMaxValue || 0,
         tickPadding: 10,
         maxOfYVal: this.props.maxOfYVal,
-        yMinMaxValues: this.props.getMinMaxOfYAxis(points, this.props.yAxisType),
+        yMinMaxValues: this.props.getMinMaxOfYAxis(
+          points,
+          this.props.yAxisType,
+          this.state.containerHeight - this.state._removalValueForTextTuncate!,
```

Comment:
>this.state.containerHeight - this.state._removalValueForTextTuncate!, [](http://example.com/codeflow?start=10&length=69)

why are these extra parameters needed. It is already part of y axis parameters to compute the final range values